lib/ally.rb

```diff
@@ -0,0 +1,43 @@
+require 'date'
+require 'oauth'
+
+module Ally
+
+  class Client
+
+    ##
+    # Creates a new Ally client
+    def initialize(client_id, secret, base_url=nil)
+      @client_id = client_id
+      @secret = secret
+      @base_url = base_url || "prod.ally.ac"
```

Comment:
Should we make `"prod.ally.ac"` the default value for the parameter instead of `nil`?